lib/source/meta/types/gimbal_type_.h

```diff
@@ -100,9 +100,11 @@ extern GBL_RESULT    GblModule_final_                  (void);
 extern GBL_RESULT    GblProperty_init_                 (GblContext* pCtx);
 extern GBL_RESULT    GblProperty_final_                (GblContext* pCtx);
 
-extern GBL_RESULT    GblSignal_init_                   (GblContext* pCtx);
-extern GBL_RESULT    GblSignal_final_                  (GblContext* pCtx);
-extern GBL_RESULT    GblSignal_removeInstance_         (GblInstance* pInstance);
+// Fix for Windows: "Redefinition with different linkage"
```

Comment:
I am *fairly* certain that you don't actually need to make these `GBL_EXPORT`, because they are internal symbols which are only ever called from within the libGimbal library itself (albeit in another translation unit, hence the forward declaration and nonstatic)... I can't verify this right now, though, so I'm going to take what you have. Will revisit later once I have repro'd this dynamic linking scenario in CI.